crates/sui/src/keytool.rs

```diff
@@ -110,11 +110,11 @@ pub enum KeyToolCommand {
         #[clap(long)]
         base64pk: String,
     },
-    /// Add a new key to sui.keystore based on the input mnemonic phrase, the key scheme flag {ed25519 | secp256k1 | secp256r1}
+    /// Add a new key to sui.keystore using either the input mnemonic phrase or a private key (from the Wallet), the key scheme flag {ed25519 | secp256k1 | secp256r1}
     /// and an optional derivation path, default to m/44'/784'/0'/0'/0' for ed25519 or m/54'/784'/0'/0/0 for secp256k1
     /// or m/74'/784'/0'/0/0 for secp256r1. Supports mnemonic phrase of word length 12, 15, 18`, 21, 24.
     Import {
-        mnemonic_phrase: String,
+        input_string: String,
         key_scheme: SignatureScheme,
```

Comment:
lgtm! i think you probably want to make the derivation-path param optional since it only applies to the mnemonics import instead of the private key import.

Reply:
Thanks for the suggestion. `derivation-path` is optional, the CLI user does not need to provide it when issuing this command.

Reply:
yeah you probably want to explicitly add to the description that it is not meant to be used for private key import